.github/workflows/binary-applications.yml

```diff
@@ -45,22 +45,16 @@ jobs:
             - name: Fetch git tags
               run: git fetch origin +refs/tags/*:refs/tags/*
 
-            - name: Restore pip cache
-              uses: actions/cache@v2
+            - name: Set up conda environment
+              uses: ./.github/actions/setup_env

```

Comment:
Is the `./` necessary?

Reply:
Yeah, it doesn't recognize it as a path otherwise. https://github.com/natcap/invest/actions/runs/1831051618